components/host-dashboard/reports-section/TransactionsOverviewSection.js

```diff
@@ -197,32 +209,27 @@ const getTransactionsBreakdownChartData = host => {
   return areas;
 };
 
-const TransactionsOverviewSection = ({ host, isLoading }) => {
-  const intl = useIntl();
-  const { locale } = intl;
-
-  const contributionStats = host?.contributionStats;
-  const expenseStats = host?.expenseStats;
-
-  const { contributionAmountOverTime } = contributionStats || 0;
-  const { expenseAmountOverTime } = expenseStats || 0;
-  const timeUnit = contributionAmountOverTime?.timeUnit;
-
+const getSeries = (host, intl) => {
+  const timeSeries = host?.hostMetricsTimeSeries || {};
   const series = [
     {
-      name: 'Contributions',
-      data: contributionAmountOverTime ? constructChartDataPoints(contributionAmountOverTime.nodes) : null,
+      name: intl.formatMessage({ id: 'Contributions', defaultMessage: 'Contributions' }),
+      data: constructChartDataPoints(timeSeries.totalReceived, node => node.kind === 'CONTRIBUTION'),
```

Comment:
I think here we have to handle the case if there's not data points within the time frame;

```suggestion
      data: constructChartDataPoints(timeSeries.totalReceived, node => node.kind === 'CONTRIBUTION') || [],
```